apps/condo/domains/marketplace/schema/Invoice.test.js

```diff
@@ -591,6 +595,210 @@ describe('Invoice', () => {
         })
     })
 
+    describe('sending push', () => {
+        test('send push after create invoice with published status', async () => {
+            const client = await makeClientWithProperty()
+
+            const unitType = FLAT_UNIT_TYPE
+            const unitName = faker.lorem.word()
+
+            const residentClient = await makeClientWithResidentUser()
+            const [resident] = await registerResidentByTestClient(
+                residentClient,
+                {
+                    address: client.property.address,
+                    addressMeta: client.property.addressMeta,
+                    unitType,
+                    unitName,
+                })
+
+            const [invoiceContext] = await createTestInvoiceContext(client, client.organization, dummyIntegration, { status: INVOICE_CONTEXT_STATUS_FINISHED })
+            const [invoice] = await createTestInvoice(client, invoiceContext, {
+                property: { connect: { id: client.property.id } },
+                unitType,
+                unitName,
+                status: INVOICE_STATUS_PUBLISHED,
+                client: { connect: { id: residentClient.user.id } },
+            })
+
+            const messages = await Message.getAll(adminClient, {
+                user: { id: residentClient.user.id },
+                type: MARKETPLACE_INVOICE_PUBLISHED_MESSAGE_TYPE,
+            })
+
+            expect(messages).toHaveLength(1)
+            expect(messages[0].meta.data.invoiceId).toEqual(invoice.id)
+            expect(messages[0].meta.data.residentId).toEqual(resident.id)
+            expect(messages[0].meta.data.url).toEqual(`${conf.SERVER_URL}/invoice/${invoice.id}`)
```

Comment:
Maybe wrap this block with `waitFor`? If I'm correct, message API executes in worker, so you need to ensure, that task will ends with expected result